contributors/design-proposals/customresources-subresources.md

```diff
@@ -94,6 +99,10 @@ If the gate is not open, the value of the new field within `CustomResourceDefini
 
 The `Scale` object is the payload sent over the wire for `/scale`. The [polymorphic `Scale` type](https://github.com/kubernetes/kubernetes/pull/53743) i.e. `autoscaling/v1.Scale` is used for the `Scale` object.
 
+Since the GVK of the `Scale` object is specified in `CustomResourceSubResourceScale`, transition to future versions (eg `autoscaling/v2.Scale`) can be done easily.
+
+Note: If `v1` is deprecated in `autoscaling/v1.Scale`, then it would be deprecated here as well.
```

Comment:
If `autoscaling/v1.Scale` is deprecated, ...

Reply:
done